### ISSUE TITLE
Unclear text in Object helpfile

### ISSUE THREAD

**Issue by celesteh** [supercollider/supercollider#5800]
The Object helpfile in SuperCollider 3.11.2 says:

> .**changed**(what ... moreArgs)
> Notify the receiver's dependants that the receiver has changed. The object making the change should be passed as theChanger.

The arguments do not correspond to the the explanation text. This reflects the argument names in the Object source.

Changing the argument names would aid clarity quite a bit, but as this may cause issues for backwards compatibility, changing the helpfile would also be a good solution.

It would be nice if information about this was expanded, as this is such a useful feature of Objects.

**Comment by telephon** [comment]
So would it be the best to write the following? `Notify the receiver's dependants that the receiver has changed. The object making the change should be passed as the first argument 'what'.`

**Comment by celesteh** [comment]
Yes, that would be much better. It would also be nice if the relationship between that and update was made more clear - especially with regards to the MoreArgs

**Comment by telephon** [comment]
yes, that would be good. Please if you like suggest something in a pull request and I'll give it a read.
